packages/push-platform-notifications-controller/tsconfig.build.json

```diff
@@ -0,0 +1,24 @@
+{
+  "extends": "../../tsconfig.packages.build.json",
+  "compilerOptions": {
+    "lib": [
+      "esnext",
+      "dom"
+    ],
+    "baseUrl": "./",
+    "outDir": "./dist/types",
+    "rootDir": "./src",
+    "skipLibCheck": true
+  },
+  "references": [
+    {
+      "path": "../base-controller/tsconfig.build.json"
+    },
+  ],
+  "include": [
+    "../../types",
+    "./src/encryption",
+    "./src/mocks",
```

Comment:
You don't have to be explicit about these — `./src` is a shortcut for `./src/**`:

``` suggestion
```

Reply:
Got it.